### PR TITLE
Update app.php

### PR DESCRIPTION
**Pull request by lucamemini**
Ugly patch for resolve issue #9 ;-)

### CODE REVIEW

File: appinfo/app.php [hunk 1]
```diff
@@ -1,4 +1,11 @@
 <?php
+## ugly patch for issue #9 https://github.com/owncloud/user_external/issues/9
+include_once('../lib/imap.php');
+include_once('../lib/smb.php');
+include_once('../lib/ftp.php');
+
+/*
```
Comment (DeepDiver1975) [inline appinfo/app.php]: just remove these lines - they are pointless

File: appinfo/app.php [hunk 2]
```diff
@@ -1,4 +1,11 @@
 <?php
+## ugly patch for issue #9 https://github.com/owncloud/user_external/issues/9
```
Comment (DeepDiver1975) [inline appinfo/app.php]: I don't think this is ugly - there is no other easy way.
You can drop this comment from my point of view

Comment (lucamemini) [reply appinfo/app.php]: ahahahah okok... for me is ugly, 
now refactoring :)

**Comment by lucamemini** [comment]
use pull https://github.com/owncloud/user_external/pull/12 this is better

**Pull request closed without merging**